portfolio/src/main/webapp/index.html

```diff
@@ -8,31 +8,32 @@
   
   <div class="container-fluid text-center padded aboutme-para">
           
-      <div id="mySidenav">
-      <a href="#" id="projects"> Projects </a>
-      <a href="#" id="about"> About </a>
-      </div>
+    <div id="mySidenav">
```

Comment:
Nit but we tend to use snake case for CSS selectors

Reply:
Just fixed it! Should I submit another pull request?

Reply:
No, you can just push the changes to this branch. 😃